packages/rich-text/README.md

```diff
@@ -29,13 +29,27 @@ create( {
 
 Create a RichText value from an `Element` tree (DOM), an HTML string or a plain text string, with optionally a `Range` object to set the selection. If called without any arguments, an empty value will be created. If `multilineTag` is provided, any content of direct children whose type matches `multilineTag` will be separated by a line separator.
 
+A value will have the following shape, which you are strongly encouraced not to modify without the use of helper functions:
```

Comment:
Reading this sentence made me wonder whether this sort of private documentation (thanks for adding it, btw) would be better off in, say, a package-specific `CONTRIBUTING.md` file. /cc @gziolo for package stuff

Reply:
Yes, this is why I hadn't added it so far, and wanted to add it when explaining how everything works together.

Reply:
Perhaps I should move it to the inline documentation for `create` for now?

Reply:
> Perhaps I should move it to the inline documentation for `create` for now?

That works!